.github/COMMIT_CONVENTION.md

```diff
@@ -0,0 +1,51 @@
+## Git Commit Message Convention
+
+> This is adapted from [Angular's commit convention](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular).
+
+#### TL;DR:
+
+Messages must be matched by the following regex:
+
+<!-- prettier-ignore -->
+```js
+/^(revert: )?(feat|fix|docs|refactor|perf|test|ci|chore|release)(\(.+\))?: .{1,50}/
+```
+
```

Comment:
or for lower humans:
type(scope): subject (footer?)
(would add that with nicer words of course 😄)